### DISCUSSION TITLE
Create a receipt (or report) for a stock transfer between warehouses

### DISCUSSION THREAD

**Discussion by redondit0** [inventree/InvenTree#7797] [Q&A] [answered]
Hello,

Is there a way to create a receipt (or report) for a stock transfer between warehouses?

For example, I have the following configuration, using it to move tools:

* General warehouse 
    * Sub-warehouse 1
    * Sub-warehouse 2

The "General warehouse" would be my house, and each Sub-warehouse is a different location where we are working.

What I'm asking is if it's possible to generate a report of the movement from the "General warehouse" to any of the sub-warehouses, so it can be printed and the recipient can check what I am sending them (Ideally, it would be something like a stock report for each location [i mean the format, not the function]. A list that can be modified from a template to include which data I want to appear).

The "General warehouse" holds the majority of the stock, but "Sub-warehouse 1" and "Sub-warehouse 2" also have some stock. I'm mentioning this because I could generate a report for the respective warehouse, but that would show the total stock of that warehouse, not just what was transferred at that moment.

Thanks!

**Comment by wolflu05** [comment, marked as answer]
This sounds like https://github.com/inventree/InvenTree/issues/7023

**Reply by redondit0** [reply]
Yup...that's pretty much what i like to do and is not possible at the moment.

**Reply by SchrodingersGat** [reply]
@redondit0 our developer time is very stretched right now - but you are welcome to work on this feature yourself, or provide funding against #7023 to speed up development :)
